cfme/tests/intelligence/reports/test_canned_corresponds.py

```diff
@@ -32,11 +32,12 @@ def test_providers_summary(soft_assert):
     presence of units and rounding."""
     path = ["Configuration Management", "Providers", "Providers Summary"]
     report = CannedSavedReport.new(path)
+    skipped_providers = {"ec2", "openstack", "redhat_network"}  # Skip cloud and network
```

Comment:
Would you add the reason these are being skipped to the comment.   That will help future maintainers if they are evaluating fi they should still be skipped.   Might need to put the comment in front of the statement then.